src/linter.cc

```diff
@@ -240,6 +243,40 @@ absl::Status CheckAliasKeyword(absl::string_view sql) {
     }).ApplyTo(sql);
 }
 
+absl::Status CheckTabCharactersUniform(absl::string_view sql,
+    const char indent, const char delimeter) {
+    bool is_indent = true;
+    const char kSpace = ' ', kTab = '\t';
+
+    for (int i=0; i<static_cast<int>(sql.size()); ++i) {
+        if ( sql[i] == delimeter ) {
+            is_indent = true;
+        } else if ( sql[i] != indent ) {
```

Comment:
please move is_indent check to the outer if, i.e.:
`esle if (is_indent && sql[i] != allowed_indent) { ...` 

This way the code is easier to read (at least for me :)